examples/CMakeLists.txt

```diff
@@ -1,15 +1,52 @@
-include_directories(${PROJECT_SOURCE_DIR}/include)
-include_directories(${PROJECT_SOURCE_DIR}/plugins)
-include_directories(${PROJECT_BINARY_DIR})
-include_directories(${PROJECT_SOURCE_DIR}/examples)
-include_directories(${PROJECT_SOURCE_DIR}/arch)
-include_directories(${PROJECT_SOURCE_DIR}/plugins/historydata)
-
-set(examples_headers
-    ${examples_headers}
-    ${PROJECT_SOURCE_DIR}/examples/common.h
-    )
-
+if(UA_EXAMPLES_STANDALONE)
```

Comment:
Instead of this additional CMake option, we can probably detect standalone if we use something similar to this:


https://github.com/CLIUtils/CLI11/blob/0387f35ec8212041fb0bae6196a922735f2fbe01/CMakeLists.txt#L26